app/representers/account_representer.rb

```diff
@@ -67,5 +70,9 @@ def credit_card_stored
     credit_card_stored?
   end
 
+  def admin_name
+    account.admins.first.try(:display_name)
```

Comment:
`account.` is not necessary 😄 you are kinda already inside the Account 😉 
So it would be just `first_admin.try!(:display_name)`, but it is even better if you delegate IMO 😄 
like:
```
delegate :display_name, to: :first_admin, allow_nil: true
alias admin_display_name display_name
```